code-challenges.js

```diff
@@ -31,38 +58,101 @@ const temperature2 = 350
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
 
+let  givenNum = (temperature1) => {
+  if(temperature1 <= 212) {
+    return `"${temperature1} is below boiling point"`
+  } 
+    else if(temperature2 >= 212) {
+    return `"${temperature2} is above boiling point"`
+  } 
+    else if(temperature3 === 212) {
+    return `"${temperature3} is at boiling point"`
+  } 
+    else {
+    return "something is wrong"
+  }
+}
+console.log(givenNum(temperature1))
+console.log(givenNum(temperature2))
+console.log(givenNum(temperature3))
+//I AM STUCK HERE-----I can't get temp3 to log at boiling point
+
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
 
+// input: combine two arrays
+// output: arrays are added together to equal length
+// create a parameter that takes two arrays of data types
+// combine length of arrays
+// return the combined length
+
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+const combinedRuns = [padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)]
+  let combinedArrays = padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)
+
+console.log(combinedArrays.length)
+
+
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Pseudo code:
 
+// input: 1 string changed into an array
+// output: 1 array reversed letters
+// create a string and change it into an array to reverse the letters
+// Use three built-in methods
+// returns the letters and numbers joined and reversed 
+
 const currentCohort = "Bravo 2023"
 // Expected output: "3202 ovarB"
 
+console.log(currentCohort.split("").reverse(" ").join(""))
+
+
 // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
 // Pseudo code:
 
+// input: 2 variables
+// output: last index of given value from array
+// create two variables with string parameters and array
+// take length of given values in array number
+// return last index of given number
+
 const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
 const givenValue1 = 42
 // Expected output: 7
 
 const givenValue2 = 10
 // Expected output: 8
 
+console.log(myNumbers.length -2)
```

Comment:
We want to make our code dynamic and not have to do some of that work like having to figure out that we want to take the length and subtract 2 or 1 from it. I think you were on the right path in your pseudocode when you mention last index of given number. Is there a method that we can use for that?